playFeature/src/main/java/com/panhuk/playfeature/PlayViewModel.kt

```diff
@@ -114,7 +111,14 @@ class PlayViewModel @Inject constructor(
     }
   }
 
-  private fun deleteThisQuestion() {
+  private fun afterCheckUpdateQuestion(){
+    deleteCurrentQuestion()
+    checkIsLastQuestion()
+    loadQuestion()
+    reInitTimer()
+  }
```

Comment:
It's not what I meant :D 

The idea is when you have a chain like this, maybe I have not that much context of the current structure, anyway, you remove the current question from the queue, then you check if it is the last item. So, it seems not necessary to call loadQuestion and reuniting timer. Did you get what I mean?